example-react-app/src/dev/component-list.tsx

```diff
@@ -0,0 +1,134 @@
+import React from "react";
```

Comment:
Let's don't generate this file. I think we could render something dynamically based on <Previews> conents